<?xml version="1.0" encoding="UTF-8"?>
<<<<<<< HEAD
<testng-results skipped="1" failed="0" total="2" passed="1">
=======
<<<<<<< HEAD
<testng-results skipped="2" failed="2" total="4" passed="0">
=======
<testng-results skipped="0" failed="2" total="2" passed="0">
>>>>>>> refs/remotes/origin/master
>>>>>>> branch 'master' of https://github.com/Pragmatic-Test-Lab/DOC_Automation.git
  <reporter-output>
  </reporter-output>
<<<<<<< HEAD
  <suite name="Default suite" duration-ms="3959" started-at="2015-01-20T03:48:17Z" finished-at="2015-01-20T03:48:21Z">
=======
<<<<<<< HEAD
  <suite name="Default suite" duration-ms="7491" started-at="2015-01-20T04:20:13Z" finished-at="2015-01-20T04:20:20Z">
=======
  <suite name="Default suite" duration-ms="8483" started-at="2015-01-20T03:33:28Z" finished-at="2015-01-20T03:33:37Z">
>>>>>>> refs/remotes/origin/master
>>>>>>> branch 'master' of https://github.com/Pragmatic-Test-Lab/DOC_Automation.git
    <groups>
    </groups>
<<<<<<< HEAD
    <test name="Default test" duration-ms="3959" started-at="2015-01-20T03:48:17Z" finished-at="2015-01-20T03:48:21Z">
      <class name="com.ptl.testcases.EditRequestCopiesTest">
        <test-method status="PASS" signature="init()[pri:0, instance:com.ptl.testcases.EditRequestCopiesTest@76b51252]" name="init" is-config="true" duration-ms="7825" started-at="2015-01-20T09:18:09Z" finished-at="2015-01-20T09:18:17Z">
=======
<<<<<<< HEAD
    <test name="Default test" duration-ms="7491" started-at="2015-01-20T04:20:13Z" finished-at="2015-01-20T04:20:20Z">
      <class name="com.ptl.testcases.GoExpregTest">
        <test-method status="PASS" signature="init()[pri:0, instance:com.ptl.testcases.GoExpregTest@4aba6039]" name="init" is-config="true" duration-ms="5728" started-at="2015-01-20T09:50:07Z" finished-at="2015-01-20T09:50:13Z">
=======
    <test name="Default test" duration-ms="8483" started-at="2015-01-20T03:33:28Z" finished-at="2015-01-20T03:33:37Z">
      <class name="com.ptl.testcases.doc244duplicateemail">
        <test-method status="PASS" signature="init()[pri:0, instance:com.ptl.testcases.doc244duplicateemail@34fccd15]" name="init" is-config="true" duration-ms="8647" started-at="2015-01-20T09:03:20Z" finished-at="2015-01-20T09:03:28Z">
>>>>>>> refs/remotes/origin/master
>>>>>>> branch 'master' of https://github.com/Pragmatic-Test-Lab/DOC_Automation.git
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
<<<<<<< HEAD
        <test-method status="PASS" signature="EditReqCopies(java.util.Hashtable)[pri:0, instance:com.ptl.testcases.EditRequestCopiesTest@76b51252]" name="EditReqCopies" duration-ms="3872" started-at="2015-01-20T09:18:17Z" data-provider="getRequestData" finished-at="2015-01-20T09:18:21Z">
=======
<<<<<<< HEAD
        <test-method status="FAIL" signature="RegNewExpTest(java.util.Hashtable)[pri:0, instance:com.ptl.testcases.GoExpregTest@4aba6039]" name="RegNewExpTest" duration-ms="6429" started-at="2015-01-20T09:50:13Z" data-provider="getExporterRegData" finished-at="2015-01-20T09:50:19Z">
=======
        <test-method status="FAIL" signature="RegNewExpTest(java.util.Hashtable)[pri:0, instance:com.ptl.testcases.doc244duplicateemail@34fccd15]" name="RegNewExpTest" duration-ms="4820" started-at="2015-01-20T09:03:28Z" data-provider="getExporterRegData" finished-at="2015-01-20T09:03:33Z">
>>>>>>> refs/remotes/origin/master
>>>>>>> branch 'master' of https://github.com/Pragmatic-Test-Lab/DOC_Automation.git
          <params>
            <param index="0">
              <value>
                <![CDATA[{Username=DCOMR00015, Runmode=Yes, COO Ref No=FTA_COUNTRY_CODE/2015/0003, No of Copies=5.0, Postal Address=87, Mayfield Road, Password=A+4bi4VX, Receipt No=7847284.0, Exporter Registration=D/COM/R/00015, Collection Method=By Post}]]>
              </value>
            </param>
          </params>
<<<<<<< HEAD
          <reporter-output>
          </reporter-output>
        </test-method> <!-- EditReqCopies -->
        <test-method status="SKIP" signature="EditReqCopies(java.util.Hashtable)[pri:0, instance:com.ptl.testcases.EditRequestCopiesTest@76b51252]" name="EditReqCopies" duration-ms="5" started-at="2015-01-20T09:18:21Z" data-provider="getRequestData" finished-at="2015-01-20T09:18:21Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Username=docdeptadmin, Runmode=No, COO Ref No=FTA_COUNTRY_CODE/2015/0003, No of Copies=5.0, Postal Address=87, Mayfield Road, Password=1qaz2wsx@, Receipt No=7847284.0, Exporter Registration=D/COM/R/00015, Collection Method=By Post}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
=======
<<<<<<< HEAD
          <exception class="org.openqa.selenium.remote.UnreachableBrowserException">
            <message>
              <![CDATA[Error communicating with the remote browser. It may have died.
Build info: version: '2.44.0', revision: '76d78cf', time: '2014-10-23 20:03:00'
System info: host: 'GMAX-PC', ip: '192.168.1.43', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0_71'
Driver info: driver.version: RemoteWebDriver]]>
=======
          <exception class="java.lang.AssertionError">
>>>>>>> branch 'master' of https://github.com/Pragmatic-Test-Lab/DOC_Automation.git
            <message>
<<<<<<< HEAD
              <![CDATA[Skipping the test]]>
=======
              <![CDATA[Exporter Registration Fail expected [true] but found [false]]]>
>>>>>>> refs/remotes/origin/master
>>>>>>> branch 'master' of https://github.com/Pragmatic-Test-Lab/DOC_Automation.git
            </message>
            <full-stacktrace>
<<<<<<< HEAD
              <![CDATA[org.testng.SkipException: Skipping the test
	at com.ptl.testcases.EditRequestCopiesTest.EditReqCopies(EditRequestCopiesTest.java:36)
=======
<<<<<<< HEAD
              <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Error communicating with the remote browser. It may have died.
Build info: version: '2.44.0', revision: '76d78cf', time: '2014-10-23 20:03:00'
System info: host: 'GMAX-PC', ip: '192.168.1.43', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0_71'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:593)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:352)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:449)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:357)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
	at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:59)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:37)
	at com.sun.proxy.$Proxy6.getText(Unknown Source)
	at com.ptl.DOC.pages.HomePage.gotoExporterRegPage2(HomePage.java:44)
	at com.ptl.testcases.GoExpregTest.RegNewExpTest(GoExpregTest.java:53)
=======
              <![CDATA[java.lang.AssertionError: Exporter Registration Fail expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.ptl.DOC.pages.ExporterRegPage.doRegisterNewExporter(ExporterRegPage.java:82)
	at com.ptl.testcases.doc244duplicateemail.RegNewExpTest(doc244duplicateemail.java:54)
>>>>>>> refs/remotes/origin/master
>>>>>>> branch 'master' of https://github.com/Pragmatic-Test-Lab/DOC_Automation.git
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to 127.0.0.1:7055 [/127.0.0.1] failed: Connection refused: connect
	at org.apache.http.impl.conn.HttpClientConnectionOperator.connect(HttpClientConnectionOperator.java:140)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:318)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:363)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:219)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:195)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:86)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:108)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:72)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:57)
	at org.openqa.selenium.remote.HttpCommandExecutor.fallBackExecute(HttpCommandExecutor.java:233)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
	at org.openqa.selenium.firefox.internal.NewProfileExtensionConnection.execute(NewProfileExtensionConnection.java:165)
	at org.openqa.selenium.firefox.FirefoxDriver$LazyCommandExecutor.execute(FirefoxDriver.java:362)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:572)
	... 33 more
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:72)
	at org.apache.http.impl.conn.HttpClientConnectionOperator.connect(HttpClientConnectionOperator.java:123)
	... 47 more
]]>
            </full-stacktrace>
<<<<<<< HEAD
          </exception> <!-- org.testng.SkipException -->
=======
<<<<<<< HEAD
          </exception> <!-- org.openqa.selenium.remote.UnreachableBrowserException -->
=======
          </exception> <!-- java.lang.AssertionError -->
>>>>>>> refs/remotes/origin/master
>>>>>>> branch 'master' of https://github.com/Pragmatic-Test-Lab/DOC_Automation.git
          <reporter-output>
          </reporter-output>
<<<<<<< HEAD
        </test-method> <!-- EditReqCopies -->
      </class> <!-- com.ptl.testcases.EditRequestCopiesTest -->
=======
        </test-method> <!-- RegNewExpTest -->
<<<<<<< HEAD
        <test-method status="FAIL" signature="RegNewExpTest(java.util.Hashtable)[pri:0, instance:com.ptl.testcases.GoExpregTest@4aba6039]" name="RegNewExpTest" duration-ms="1046" started-at="2015-01-20T09:50:19Z" data-provider="getExporterRegData" finished-at="2015-01-20T09:50:20Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{COO Mobile=0723994254, Contact Person=Harry Jaawreerten, Official Address=107, New kendy rd, Mabaso, Username=docdeptadmin, Fax=0192699459, Runmode=yes, Tel=0112999922, Company Name=VPELintar, Password=1qaz2wsx@, COO Email=hay2j29@gmail.com, COO Name=Harry erereeerria, Business RegNo=LSA95334, Contact Email=hrr9332@gmail.com}]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.remote.UnreachableBrowserException">
            <message>
              <![CDATA[Error communicating with the remote browser. It may have died.
Build info: version: '2.44.0', revision: '76d78cf', time: '2014-10-23 20:03:00'
System info: host: 'GMAX-PC', ip: '192.168.1.43', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0_71'
Driver info: driver.version: FirefoxDriver]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Error communicating with the remote browser. It may have died.
Build info: version: '2.44.0', revision: '76d78cf', time: '2014-10-23 20:03:00'
System info: host: 'GMAX-PC', ip: '192.168.1.43', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0_71'
Driver info: driver.version: FirefoxDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:593)
	at org.openqa.selenium.firefox.FirefoxDriver.access$0(FirefoxDriver.java:1)
	at org.openqa.selenium.firefox.FirefoxDriver$1$1.implicitlyWait(FirefoxDriver.java:219)
	at com.ptl.testcases.GoExpregTest.RegNewExpTest(GoExpregTest.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to 127.0.0.1:7055 [/127.0.0.1] failed: Connection refused: connect
	at org.apache.http.impl.conn.HttpClientConnectionOperator.connect(HttpClientConnectionOperator.java:140)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:318)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:363)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:219)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:195)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:86)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:108)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:72)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:57)
	at org.openqa.selenium.remote.HttpCommandExecutor.fallBackExecute(HttpCommandExecutor.java:215)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
	at org.openqa.selenium.firefox.internal.NewProfileExtensionConnection.execute(NewProfileExtensionConnection.java:165)
	at org.openqa.selenium.firefox.FirefoxDriver$LazyCommandExecutor.execute(FirefoxDriver.java:362)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:572)
	... 27 more
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:72)
	at org.apache.http.impl.conn.HttpClientConnectionOperator.connect(HttpClientConnectionOperator.java:123)
	... 41 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.remote.UnreachableBrowserException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RegNewExpTest -->
        <test-method status="SKIP" signature="ReadExporterRegistrationNumberTest()[pri:0, instance:com.ptl.testcases.GoExpregTest@4aba6039]" name="ReadExporterRegistrationNumberTest" duration-ms="0" started-at="2015-01-20T09:50:20Z" depends-on-methods="com.ptl.testcases.GoExpregTest.RegNewExpTest" finished-at="2015-01-20T09:50:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ReadExporterRegistrationNumberTest -->
        <test-method status="SKIP" signature="RegisteredExporterLoginTest()[pri:0, instance:com.ptl.testcases.GoExpregTest@4aba6039]" name="RegisteredExporterLoginTest" duration-ms="0" started-at="2015-01-20T09:50:20Z" depends-on-methods="com.ptl.testcases.GoExpregTest.RegNewExpTest" finished-at="2015-01-20T09:50:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RegisteredExporterLoginTest -->
      </class> <!-- com.ptl.testcases.GoExpregTest -->
=======
        <test-method status="FAIL" signature="RegNewExpTest(java.util.Hashtable)[pri:0, instance:com.ptl.testcases.doc244duplicateemail@34fccd15]" name="RegNewExpTest" duration-ms="3614" started-at="2015-01-20T09:03:33Z" data-provider="getExporterRegData" finished-at="2015-01-20T09:03:37Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{COO Mobile=0723994254, Contact Person=Harry Jaawreerten, Official Address=107, New kendy rd, Mabaso, Username=docdeptadmin, Fax=0192699459, Runmode=yes, Tel=0112999922, Company Name=VPELintar, Password=1qaz2wsx@, COO Email=hay2j29@gmail.com, COO Name=Harry erereeerria, Business RegNo=LSA95334, Contact Email=hrr9332@gmail.com}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Exporter Registration Fail expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Exporter Registration Fail expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.ptl.DOC.pages.ExporterRegPage.doRegisterNewExporter(ExporterRegPage.java:82)
	at com.ptl.testcases.doc244duplicateemail.RegNewExpTest(doc244duplicateemail.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RegNewExpTest -->
      </class> <!-- com.ptl.testcases.doc244duplicateemail -->
>>>>>>> refs/remotes/origin/master
>>>>>>> branch 'master' of https://github.com/Pragmatic-Test-Lab/DOC_Automation.git
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
